### PR TITLE
Minor cleanup around the codebase

### PR DESCRIPTION
**Pull request by RaitoBezarius**
- Mainly, unused imports.
- Unexpected statements in code, e.g. using `os` when not imported.
- Moving `requirements` into a proper folder.

### CODE REVIEW

**Comment by vendethiel** [comment]
If you move the requirements file, you probably should update the readme

**Comment by RaitoBezarius** [comment]
@vendethiel Good catch!

File: mangaki/mangaki/utils/knn.py [hunk 1]
```diff
@@ -94,7 +97,12 @@ def get_common_traits(self, my_username, username):
             if abs(my) >= 1 and abs(their) >= 1:
                 print('%d.' % rank, works[work_id].title, my, their, '=', my * their)
 
-    def fit(self, X=[], y=[], all_dataset=False):
+    def fit(self, X=None, y=None, all_dataset=False):
+        if X is None:
+            X = []
+        if y is None:
+            y = []
```
Comment (jilljenn) [inline mangaki/mangaki/utils/knn.py]: Why?

Comment (vendethiel) [inline mangaki/mangaki/utils/knn.py]: Python only evaluates default arguments **once**.

```python
>>> def f(arg = []):
...   arg.append(3)
...   return arg
>>> f()
[3]
>>> f()
[3, 3]
>>> f()
[3, 3, 3]
```

Comment (RaitoBezarius) [inline mangaki/mangaki/utils/knn.py]: `list` are mutable in Python.

Comment (vendethiel) [inline mangaki/mangaki/utils/knn.py]: I'm not sure why X is uppercased however :).

Comment (RaitoBezarius) [inline mangaki/mangaki/utils/knn.py]: Ask @jilljenn — my understanding was: `X` represents a matrix (in maths, we often uppercase them, so it was "domain specific code style" that I tried to kept.)

**Comment by jilljenn** [comment]
So, you just need to edit the README?

Minor question: Why “RaitoBezarius committed with RaitoBezarius 3 days ago”?

**Comment by RaitoBezarius** [comment]
@jilljenn Right.
The answer to the minor question is: I was trying a new git wrapper (the `PyCharm` one) and it has this nice checkbox of `Signed off by`, which I guess would translate to "committed with …".

So I signed off my commits by myself. Indeed, this is useless.

File: README.md [hunk 2]
```diff
@@ -45,7 +45,7 @@ environnement virtuel, ce qui est fait par les commandes ci-dessous.
     
     python3 -m venv venv --system-site-packages
     . venv/bin/activate
-    pip install -r requirements-dev.txt # Si installation d'une instance de développement, sinon utilisez requirements.txt
+    pip install -r requirements/dev.txt # Si il s'agit d'une instance de développement, sinon utilisez requirements/production.txt
```
Comment (jilljenn) [inline README.md]: Si il => S'il

**Pull request merged**